packages/jaeger-ui/src/middlewares/index.js

```diff
@@ -45,4 +47,29 @@ export const historyUpdateMiddleware = store => next => action => {
   return next(action);
 };
 
+export const transformOTLPMiddleware = store => next => action => {
```

Comment:
is it supposed to be called from somewhere?

Reply:
The middleware is applied to listen to actions in configure-store.js, hence these middelwares are triggered when the particular store is dispatched. I don't have to call them explicitly anywhere

Reply:
is it the fact that it's exported what makes it included/called?

Reply:
yes, and here https://github.com/jaegertracing/jaeger-ui/blob/60e5ce43746c42e4a8cf4cbb77ff2eeed2a0c46c/packages/jaeger-ui/src/utils/configure-store.js#L41-L45 

the middle wares are imported and applied